drivers/r2dbc-driver/build.gradle

```
@@ -8,9 +8,8 @@ archivesBaseName = 'sqldelight-r2dbc-driver'
 
 dependencies {
   api project(':runtime')
-  implementation libs.r2dbc
-  implementation libs.kotlin.coroutines.core
-  implementation libs.kotlin.coroutines.reactive
+  api libs.r2dbc
+  api libs.kotlin.coroutines.reactive
```

```suggestion
  implementation libs.kotlin.coroutines.reactive
```
Even if someone is most likely using coroutines with this driver, I don't think it makes sense to force exposing this dependency on anyone.

---

Changing this line was the main/only motiviation for this PR :D 
Without `awaitSingle()` you are unable to get a connection in a comfortable way.